### PR TITLE
Profile for LPAR : lnxocp04

### PR DESCRIPTION
**Pull request by phani2898**
Added a tunnel profile file for new yellow zone LPAR lnxocp04 for HyperShift PROW CI use-case.

### CODE REVIEW

**Comment by deepsm007** [comment]
Cc @jaypoulz

**Comment by jaypoulz** [comment]
@phani2898 I've asked @tvardema to review and get back to you. :)

File: libvirt/tunnel/profile_lnxocp04.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+profile:
+  arch: "s390x"
+  cluster_capacity: 0
```
Comment (tvardema) [inline libvirt/tunnel/profile_lnxocp04.yaml:3]: Is this file just a placeholder for an LPAR or should this cluster capacity be updated to something greater than zero?

File: libvirt/tunnel/profile_lnxocp04.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  environment: "bastion-z"
+libvirt:
+  bastion-port: 16512
+  target-port: 16509
+http:
+  bastion-port: 8083
+  target-port: 80
+https:
+  bastion-port: 8446
+  target-port: 443
```
Comment (tvardema) [inline libvirt/tunnel/profile_lnxocp04.yaml:14]: The other tunnel profiles have bastion port references.  Are they unneeded for this lpar?

**Comment by jaypoulz** [comment]
Some additional context.
1. You're going to need to run the generate ports script in that directory once you have a valid profile.
2. The bastion port is something we added in Deep's era so that we could ssh to a node that can see and connect to the clusters from within the yellow zone. If we want to omit them, we'll want to check in openshift/release to see if we actually reference them in stuff like the must-gather steps that try to pull info from the cluster nodes.

**Comment by jaypoulz** [comment]
Also, it looks to me like the "bastion" addresses we use in the tunnel script are really just the bootstrap node:
https://github.com/multi-arch/ocp-remote-ci/blob/master/libvirt/tunnel/tunnel.sh#L53C1-L53C47

File: libvirt/tunnel/profile_lnxocp04.yaml (hunk 1 above)
Comment (phani2898) [reply libvirt/tunnel/profile_lnxocp04.yaml:3]: Hey @tvardema , In this LPAR we don't create an entire cluster, we just create 2 VMs which will be attached as compute nodes to the Hosted Control Plane as part of HyperShift agent based CI. So kept the cluster_capacity to 0

File: libvirt/tunnel/profile_lnxocp04.yaml (hunk 2 above)
Comment (phani2898) [reply libvirt/tunnel/profile_lnxocp04.yaml:14]: The bastion port references for other LPARs are just nothing bootstrap nodes required for the cluster creation on those LPARs. As we are nor creating any clusters here we would not require any bootstrap/bastion port references.

**Review by tvardema — APPROVED** [review]

**Pull request merged**